docs/source/install.md

```diff
@@ -0,0 +1,147 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.13.7
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# Install
+
+EvalML is available for Python 3.8 with experimental support 3.9. It can be installed with pip or conda.
```

Comment:
```suggestion
EvalML is available for Python 3.8 with experimental support 3.9. It can be installed from [pypi](https://pypi.org/project/evalml/), [conda-forge](https://anaconda.org/conda-forge/evalml), or from [source](https://github.com/alteryx/evalml).
```